src/items/controllers/item.controller.ts

```diff
@@ -21,4 +21,9 @@ export class ItemsController {
   create(@Body() itemRequestDto: ItemRequestDto): Promise<Item> {
     return this.itemService.create(itemRequestDto);
   }
+
+  @Post('/price/:itemId')
```

Comment:
I'd suggest yo use the rest nomenclature, {entity}/{id}/{action}, meaning your route would look something like 'http://domain/v1/items/123456/price',

Reply:
Yes, I just realized